Scripts/buildAndRun.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+# build
+#  - add 'cci' tag
+#  - path: . (current directory)
+docker build -t cci .
+
+if [ $? -eq 0 ]
+then
+  echo "Successfull build"
+
+# sound effect after build
+  echo '\007'
+  
+# run
+#  - set environment variabes with '-e'
+#    - set debugMode=true for debug messages
+#  - port: localhost 8080 -> cci 8081
+#  - docker image tag: cci
+  docker run -it \
+  -e port="8080" \
+  -e slackToken="SLACKTOKEN" \
+  -e circleCiTokens="CIRCLECITOKENS" \
+  -e circleCiVcs="github" \
+  -e circleCiProjects="4dmotion-ios@cci" \
+  -e circleCiCompany="CIRCLECICOMPANY" \
+  -e githubSecret="GITHUBSECRET" \
+  -e githubAppId="GITHUBAPPID" \
+  -e githubPrivateKey="GITHUBPRIVATEKEY" \
+  -e youtrackURL="YOUTRACKURL" \
+  -e youtrackToken="YOUTRACKTOKEN" \
+  -e debugMode="true" \
+  -p 8080:8081 \
+  cci
+else
+  echo "Build has failed"
+# sound effect after build
+  echo '\007'
+
+fi
+
+
+# sample call:
```

Comment:
Unnecessary because `slack.sh` does this job? Maybe we should refer to slack.sh, instead